go.mod

```
@@ -0,0 +1,5 @@
+module tarsnapmanager
```

I think module names should follow the usual import path, so this should be

```suggestion
module github.com/jamesog/tarsnapmanager
```